website/content/docs/auth/kubernetes.mdx

```diff
@@ -68,7 +78,10 @@ management tool.
     $ vault auth enable kubernetes
     ```
 
-1.  Use the `/config` endpoint to configure Vault to talk to Kubernetes. Use `kubectl cluster-info` to validate the Kubernetes host address and TCP port. Kubernetes 1.21+ clusters may require setting the service account `issuer`, [as described here](/docs/auth/kubernetes#discovering-the-service-account-issuer). For the list of available configuration options, please see the [API documentation](/api/auth/kubernetes).
+1.  Use the `/config` endpoint to configure Vault to talk to Kubernetes. Use
+`kubectl cluster-info` to validate the Kubernetes host address and TCP port.
+For the list of available configuration options, please see the
+[API documentation](/api/auth/kubernetes).
 
     ```text
```

Comment:
Good spot. This was annoying to figure out, but the fix was to indent the first paragraph of text for this bullet so that the note gets properly parsed as a paragraph within the bullet. I'm still confused how I broke it in the first place though.